app/controllers/users_controller.rb

```diff
@@ -107,6 +108,29 @@ def signup_prep
     @page_title = 'Sign Up'
   end
 
+  def og_data
+    data = {
+      url: user_url(@user),
+      title: @user.username,
+    }
+    if @user.avatar.present?
+      data[:image] = {
+        src: @user.avatar.url,
+        width: '75',
+        height: '75',
+      }
+    end
+
+    board_ids = BoardAuthor.where(user_id: @user.id, cameo: false).pluck('distinct board_id')
+    arel = Board.arel_table
+    where = arel[:creator_id].eq(@user.id).or(arel[:id].in(board_ids))
+    boards = Board.where(where).ordered
+
+    data[:description] = generate_short(boards.pluck(:name) * ', ')
```

Comment:
Perhaps we should say "Continuities:" before this? I'm not sure – it might be clear.